mercury_app/utils.py

```diff
@@ -632,3 +638,34 @@ def get_db_transaction_by_merchandise(merchandise):
         return transaction_query
     except Exception:
         return None
+
+
+class EventAccessMixin():
+    def get_event(self):
+        event = get_object_or_404(
+            Event,
+            eb_event_id=self.kwargs['event_id'],
+        )
+        org_id = UserOrganization.objects.filter(
+            user=self.request.user).values_list('organization', flat=True)
+        if event.organization.id not in org_id:
```

Comment:
It would be great if you can raise this exception asap or at least before any DB transaction

Reply:
I need to know the organizations of the user before raise the exception